### ISSUE TITLE
Could add warn for unused parameters in config?

### ISSUE THREAD

**Issue by ykuc** [doublecloud/transfer#138]

**Comment by ykuc** [comment]
It marks all fields for yt dest in unused fields.
Is this result ok?

```yaml
dst:
  type: yt
  params: |
    {
      "path": "//home/cdc",
      "cluster": "yt-backend:80",
      "cellbundle": "default",
      "primarymedium": "default"
    }
```

```bash 
./binaries/trcli validate --transfer=./examples/pg2yt/transfer_cdc_embed.yaml

2024-12-21T18:20:48.228+0100    INFO    trcli/config/config.go:147      config for: pg source has [IgnoreUnknownTables SnapshotDegreeOfParallelism DBLogEnabled ChunkSize IsHomo TLSFile DesiredTableSize BatchSize KeeperSchema IgnoreUserTypes EmitTimeTypes ConnectionID DBTables SecurityGroupIDs Host EnableTLS SubNetworkID CollapseInheritTables PreSteps PgDumpCommand SlotID UseFakePrimaryKey ExcludedTables SlotByteLagLimit UsePolling AutoActivate PostSteps ExcludeDescendants SnapshotSerializationFormat ShardingKeyFields Cluster MaxBufferSize NoHomo] unset fields
2024-12-21T18:20:48.228+0100    INFO    trcli/config/config.go:162      config for: yt destination has [primarymedium cellbundle cluster path] unused fields
2024-12-21T18:20:48.228+0100    INFO    trcli/config/config.go:165      config for: yt destination has [Model _pushWal] unset fields
2024-12-21T18:20:48.228+0100    INFO    trcli/validate/validate.go:32   pg 👌source config
2024-12-21T18:20:48.228+0100    INFO    trcli/validate/validate.go:38   yt 👌destination config
```

**Comment by laskoviymishka** [comment]
That's a separate issue specifically with YT Provider, will be fixed here: #151
